src/pages/questionPage.js

```diff
@@ -9,38 +9,47 @@ import {
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
-import { createScoreElement } from '../views/scoreView.js'
 
 export const initQuestionPage = () => {
-  console.log(quizData)
+  console.log(quizData);
   const userInterface = document.getElementById(USER_INTERFACE_ID);
   userInterface.innerHTML = '';
 
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-  const correctAnswer = currentQuestion.correct
+  const correctAnswer = currentQuestion.correct;
 
   const questionElement = createQuestionElement(currentQuestion.text);
-
   userInterface.appendChild(questionElement);
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
   //create element to show the score
```

Comment:
removed scoreElement is in the newer version of the code as it was added in the views, we could remove the created element on lines 28 and 29 and keep lines 30 and 31, and also to put the previous declaration of the scoreElement